### ISSUE TITLE
Takes at least two tries to read the CD on Data Expert EXP 4349

### ISSUE THREAD

**Issue by ualaa1** [86Box/86Box#4969]
### What happened?

CD driver used:
[CD Driver.zip](https://github.com/user-attachments/files/17789427/CD.Driver.zip)
Config, HDD, NVR: [Configuration.zip](https://github.com/user-attachments/files/17789430/Configuration.zip)
This HDD only contains the CD drivers, no other drivers (Sound, mouse etc.) are present, system is MS DOS 6.22. Have also tested configuring CD speed as well as configuring IDE channel from 0:1 to 1:0, and changing HD controllers. It occurs on any CD inserted into the CD drive
Error still occurs
Screenshot:
![Monitor_1_20241117-002018-467](https://github.com/user-attachments/assets/ee86934d-08d4-4911-aa0d-731a86886a6e)

### Configuration file

```ini
[General]
vid_renderer = qt_software
host_cpu = Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
emu_build_num = 6130
uuid = 8ec07f16-0be1-5a53-9f7b-3745a1048a10

[Machine]
machine = exp4349
cpu_family = pentium_p24t
cpu_speed = 83333333
cpu_multi = 2.5
cpu_use_dynarec = 1
fpu_softfloat = 0
time_sync = local
fpu_type = internal
mem_size = 32768

[Video]
gfxcard = px_trio64_vlb

[Input devices]
mouse_type = msserial

[Sound]
fm_driver = nuked
sndcard = sbawe32
midi_device = fluidsynth

[Network]
net_01_link = 0
net_02_link = 0
net_03_link = 0
net_04_link = 0

[Storage controllers]
hdc = ide_vlb_2ch
cassette_mode = load

[Sound Blaster AWE32 #1]
base = 0220
emu_base = 0620
base401 = 0000
irq = 5
dma = 1
dma16 = 5
onboard_ram = 28672
opl = 1
control_pc_speaker = 0
receive_input = 1
receive_input401 = 0

[FluidSynth]
sound_font = C:/Users/19099/Desktop/em/dos3/SC-55.SoundFont.v1.2b.sf2
output_gain = 20
chorus = 1
chorus_voices = 3
chorus_level = 20
chorus_speed = 30
chorus_depth = 80
chorus_waveform = 0
reverb = 1
reverb_room_size = 20
reverb_damping = 0
reverb_width = 5
reverb_level = 90
interpolation = 2

[Hard disks]
hdd_01_parameters = 63, 16, 4063, 0, ide
hdd_01_fn = Bighd.vhd
hdd_01_speed = ramdisk
hdd_01_vhd_blocksize = 4096
hdd_01_ide_channel = 0:0

[Floppy and CD-ROM drives]
fdd_01_type = 35_2ed
fdd_01_turbo = 1
fdd_02_type = 35_2ed
fdd_02_turbo = 1
cdrom_01_speed = 72
cdrom_01_parameters = 1, atapi
cdrom_01_ide_channel = 0:1
fdd_01_image_history_01 = CD Driver.IMA
cdrom_01_type = 86BOX_CD-ROM_1.00
cdrom_01_image_path = sbawe32.cue
```

### Operating system

Windows 11

### CPU

Intel Core i7-9750H 

### 86Box version

4.2.1

### Build architecture

Windows - x64 (64-bit)

### Build type

- [ ] New recompiler
- [ ] Debug build

### Download source

Official website (Jenkins, GitHub)

**Comment by NebularNerd** [comment]
I've not played with 86Box but from DosBox-X I found an old driver which I used to use for system builds in the 90's to be pretty reliable. It might be worth grabbing the IDE-CD zip file from the comment linked below and giving that a try. There's no installer or anything, but if you copy it to your emulated C: drive and edit the config.sys to look at that driver instead of the driver from your floppy image, it may work. 🙂

https://github.com/joncampbell123/dosbox-x/issues/3752#issuecomment-1253428883
